kustomize/README.md

```diff
@@ -0,0 +1,228 @@
+## INSTALLING TERRASCAN IN A KUBERNETES CLUSTER USING KUSTOMIZE
+
+This guide will help you install terrascan server inside your kubernetes cluster, as per your specific use case.
+We have covered four use cases in the sections below:
+
+  ##### Deploying Terrascan Server
+  Terrascan operating in http server mode.
+
+  #### Deploying Terrascan Server in TLS Mode
+  Terrascan operating in https server mode. This deployment is also a foundation for the terrascan webhook setup.
+
+  #### Deploying Terrascan Server for Remote Repository Scan
+  Terrascan in https server mode, with capability to scan remote repositories. This deployment can be used by an
+  argocd pre-sync hook that sends remote repository scan requests to the server.
+
+  #### Setting Up Terrascan Webhook
+  A Kubernetes Validating Webhook, that safeguards your cluster by denying the creation of kubernetes resources that
+  can cause potential security violations.
+
+### PRE-REQUISITE
+1. Make sure you have required access on the kubernetes cluster to create and update the following resources:
+
+  - Secrets
+  - Configmaps
+  - Deployments
+  - Services
+  - ValidatingWebhookConfiguration (only if you're aiming to deploy the webhook as well)
+
+  **If it is not a production level cluster, you probably do have the required access.**
+
+2. Make sure you have `kubectl`, `kustomize` and `openssh` installed on your local machine.
+
+3. Make sure you replace `<TERRASCAN_NAMESPACE>` placeholder with your target namespace where you to want to deploy the
+terrascan server. The string replacement will be required in the following files:
+
+  - `base/kustomization.yaml`
+  - `server/kustomization.yaml`
+  - `server-tls/kustomization.yaml`
+  - `server-tls/certs/domain.cnf` (that is generated in step 1 of `Deploying Terrascan Server in TLS Mode` section)
+  - `webhook/kustomization.yaml` (only if you're aiming to deploy the webhook as well)
+  - `webhook/validating-webhook.yaml` (only if you're aiming to deploy the webhook as well)
```

Comment:
Add server-remote-repo-scan/kustomization.yaml also, we have to replce namespace in that file as well